src/models/blockchain/mod.rs

```diff
@@ -53,7 +53,7 @@ pub enum TransactionType {
 	/// EVM transaction
 	EVM(evm::EVMTransaction),
 	/// Stellar transaction
-	Stellar(stellar::StellarTransaction),
+	Stellar(Box<stellar::StellarTransaction>),
```

Comment:
Why are we boxing this? Was it complaining about size?

Reply:
Yep it did, newer clippy version atleast with 1.87.0. Boxing it makes sense anyways to avoid the size issues.